pippo-session-parent/pippo-session/src/main/java/ro/pippo/session/SessionData.java

```diff
@@ -28,112 +28,31 @@
  *
  * @author Decebal Suiu
  */
-public class SessionData implements Serializable {
+public interface SessionData extends Serializable {
 
-    public static final int DEFAULT_MAX_INACTIVE_INTERVAL_SECONDS = 30 * 60; // 30 minutes
+    int DEFAULT_MAX_INACTIVE_INTERVAL_SECONDS = 30 * 60; // 30 minutes
```

Comment:
Move `DEFAULT_MAX_INACTIVE_INTERVAL_SECOND` in `DefaultSessionData` (I think that is the only place where is used)